client/add/add.ctrl.js

```diff
@@ -87,13 +89,16 @@ module.exports = class Add {
       .tap(box => {
         this.addedToast(`Box '${box.name}' created successfully`, 'View', `/box/${box.id}`);
       })
-      .then(res => this.$scope.$apply(this.boxes.push(res)))
+      .tap(box => this.$scope.$apply(this.boxes.push(box)))
+      .then(box => this.$scope.$apply(this.boxesById[box.id].push(box)))
```

Comment:
It looks like this was already here before, but is it intended that this will call `$scope.$apply` on a number?